bin/install-qa-check.d/60pkgconfig

```diff
@@ -103,26 +103,38 @@ pkgconfig_check() {
 
 	# Skip the check if QA_PKGCONFIG_VERSION is set to empty string.
 	if [[ -n ${QA_PKGCONFIG_VERSION} ]]; then
-		local all_bad=yes
+		local pms_ver_re="^([0-9]+(\.[0-9]+)*)([a-z]?)((_(alpha|beta|pre|rc|p)[0-9]*)*)(-r[0-9]+)?$"
 		local -A bad_files
+
+		local is_pms_ver=false
+		if [[ ${QA_PKGCONFIG_VERSION} =~ ${pms_ver_re} ]] ; then
+			# Ensure that ver_test is available.
+			inherit eapi7-ver
```

Comment:
This will die on newer EAPIs

Reply:
I went with `[[ $(type -f ver_test) == function ]] || inherit eapi7-ver` (untested)